frontend/app/views/fragments/offer/callout.scala.html

```diff
@@ -0,0 +1,12 @@
+@(imgSrc: String, text: String = "")
+
+@import views.support.Asset
+
+<div class="offer-letterbox">
```

Comment:
Is letterbox the ful-width images? I don't think this should be coupled to an offer, a generic letterbox component would be preferable.